documentation20/en/12.taos-sql/docs.md

```diff
@@ -1,8 +1,8 @@
 # TAOS SQL
 
-TDengine provides a SQL-style language, TAOS SQL, to insert or query data. To read through this document, you should have some basic understanding about SQL.
+TDengine provides a SQL-style language, TAOS SQL, to insert or query data, and the document supports other common tips. To finish this document, readers should have basic understanding about SQL.
```

Comment:
"this document support".

Reply:
I think we should keep the "read through". it has clearer meaning than "finish".